src/index.html

```diff
@@ -3,33 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello</title>
+    <title>Leaderboard</title>
     <script src="https://kit.fontawesome.com/05c5795d1a.js" crossorigin="anonymous"></script>
 <script defer src="main.js"></script></head>
 <body>
-    <h1>Leaderboard</h1>
-    <div class="big-container">
-        <div class="container-left">
-            <div class="c1r1">
-                <h2>Recent Scores</h2>
-                <button>Refresh</button>
-            </div>
-            <div class="c1r2">
-                <ul id="leaderboard">
-                    <li class="scorer">N1</li>
-                    <li class="scorer">N2</li>
-                    <li class="scorer">N3</li>
-                    <li class="scorer">N4</li>
-                    <li class="scorer">N5</li>
-                </ul>
-            </div>
-        </div>
-        <div class="container-right">
-            <h2>Add your score</h2>
-            <input type="text" id="name" placeholder="Name">
-            <input type="text" id="scorde" placeholder="Score">
-            <button>Add score</button>
-        </div>
-    </div>
+	<h1>Leaderboard</h1>
+	<div class="big-container">
+			<div class="container-left">
+					<div class="c1r1">
+							<h2>Recent Scores</h2>
+							<button id="refreshbtn">Refresh</button>
+					</div>
+					<div class="c1r2">
+							<ul id="leaderboard">
+							</ul>
+					</div>
+			</div>
+			<div class="container-right">
+					<h2>Add your score</h2>
+					<input type="text" id="name" placeholder="Name">
+					<input type="text" id="score" placeholder="Score">
+					<button id="addbtn">Add score</button>
+			</div>
+	</div>
 </body>
```

Comment:
- [OPTIONAL] Be careful with blank lines and indentation. Do not add blank lines, spaces, or indentations without a reason. For readability, add blank lines to separate large or logical code blocks and add two spaces of indentation. Do not use the tab key 👍🏻

Reply:
I'm not sure why in Github it appears like that @BenMukebo. If you clone the repo it doesn't, I can't change it from local.